src/app/modules/time-clock/components/entry-fields/entry-fields.component.spec.ts

```diff
@@ -155,12 +213,37 @@ describe('EntryFieldsComponent', () => {
   it('when a start hour is updated, then dispatch UpdateActiveEntry', () => {
     component.activeEntry = entry ;
     component.setDataToUpdate(entry);
+    const newHour = moment().format('HH:mm:ss');
```

Comment:
let's use `newTime` or `updatedTime` instead of `newHour`

Reply:
Ok, I change to `updatedTime`